lms/djangoapps/certificates/management/commands/cert_generation.py

```diff
@@ -0,0 +1,75 @@
+"""
+Management command to generate course certificates for one or more users in a given course run.
+"""
+
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from lms.djangoapps.certificates.generation_handler import generate_certificate_task
+
+User = get_user_model()
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command to generate course certificates for one or more users in a given course run.
+
+    Example usage:
+    ./manage.py lms cert_generation -u 123 456 -c course-v1:edX+DemoX+Demo_Course
+    """
+
+    help = """
+    Generate course certificates for one or more users in a given course run.
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-u', '--user',
+            nargs="+",
+            metavar='USER',
+            dest='user',
+            required=True,
+            help='user_id or space-separated list of user_ids for whom to generate course certificates'
+        )
+        parser.add_argument(
+            '-c', '--course-key',
+            metavar='COURSE_KEY',
+            dest='course_key',
+            required=True,
+            help="course run key"
+        )
+
+    def handle(self, *args, **options):
+        if not options["user"]:
+            raise CommandError("You must specify a list of users")
+
+        course_key = options['course_key']
```

Comment:
I believe this is protected also by the argument being required. 

That being said this will throw an exception if the key is not in the dictionary. If you are concerned the key is not in the dictionary you should use the `.get()` method. It returns none if the key is not present rather than throwing an exception like a Dictionary access will for a missing key.